ceph_deploy/tests/test_mon.py

```diff
@@ -20,6 +21,45 @@ def make_fake_conn(receive_returns=None):
     return conn
 
 
+class TestGetMonInitialMembers(object):
+
+    def list_and_size_checker(self, chk_item, flag):
+        if isinstance(chk_item, list):
+            if len(chk_item) == 1 and flag == 'single':
+                return True
+            if len(chk_item) > 1 and flag == 'multiple':
+                return True
+        else:
+            raise AssertionError('Return value is invalid! (isList:%s, List size:%d)' %
+                                    (isinstance(chk_item, list), len(chk_item)))
+
+    def test_assert_if_mon_none_and_empty_True(self):
+        cfg = make_fake_conf()
+        with pytest.raises(exc.NeedHostError):
+            mon.get_mon_initial_members(Mock(), True, cfg)
+
+    def test_return_if_mon_none_and_empty_false(self):
+        cfg = make_fake_conf()
+        mon_initial_members = mon.get_mon_initial_members(Mock(), False, cfg)
+        if mon_initial_members is None:
+            return True
+        raise AssertionError("Return value is invalid! (Should be None)")
```

Comment:
Normally with unit tests you would just assert that the return is what you know it should be.

So I think this can just be:

assert mon_initial_members == None

It's actually unusual to raise your own AssertionError in tests.  You instead assert what you want the test to result it.

http://pytest.org/latest/assert.html may help